src/public/app/services/toast.js

```diff
@@ -2,15 +2,18 @@ import ws from "./ws.js";
 import utils from "./utils.js";
 
 function toast(options) {
-    const $toast = $(`<div class="toast" role="alert" aria-live="assertive" aria-atomic="true">
-    <div class="toast-header">
-        <strong class="mr-auto"><span class="bx bx-${options.icon}"></span> <span class="toast-title"></span></strong>
-        <button type="button" class="ml-2 mb-1 close" data-dismiss="toast" aria-label="Close">
-            <span aria-hidden="true">&times;</span>
-        </button>
-    </div>
-    <div class="toast-body"></div>
-</div>`);
+    const $toast = $(
```

Comment:
Can you double-check the toast width (e.g. when deleting a note)? I think the new one is quite a bit larger than what it used to be.

Reply:
It's do a bit larger. But I think it's should be fine, since upgrading cannot always have the exact same layout with the old version. And it doesn't affect user experience I think.